perma_web/perma/celery_tasks.py

```diff
@@ -1350,6 +1352,29 @@ def format_filesize(i):
         wacz_size = 0
 
     formatted_wacz_size = format_filesize(wacz_size)
+    if wacz_size:
+        # retrieve S3's md5 hash of file it has
+        remote_hash = default_storage.connection.Object(
+            bucket_name=default_storage.bucket_name,
+            key=os.path.join(settings.MEDIA_ROOT, link.warc_storage_file())
+        ).e_tag.strip('"')
+
+        # calculate the same thing for the warc inside the wacz
+        blocksize = 2 ** 20
+        m = hashlib.md5()
+        with ZipFile(wacz_path) as zip_file:
+            with zip_file.open(f"archive/{link.guid}.warc.gz") as embedded_warc:
```

Comment:
> Have we decided to use {link.guid}.warc.gz as the WARC filename instead of e.g. data.warc.gz? I am not very familiar with how people interact with WACZs aside from reading the spec; is this in line with expected file naming patterns?

So you can have lots of WARCs inside a single WACZ; they can be named according to whatever convention; data.warc.gz is just what Scoop does, probably because it's filename used as an example in the spec. 

Since our WARCs are currently named `{link.guid}.warc.gz`, if we pass them to js-wacz for conversion, it just keeps that untouched.

I couldn't think of any reason to switch; I asked Matteo if he could think of any reason to switch and he couldn't think of one either. 

I can see it both ways:
- if we introduce heterogeneity into our WACZ collection, then we'll have to account for that in the future, say, if we ever want to programmatically extract WARCs from WACZs... which could be annoying
- on the other hand, this distinction would make it possible to see, from the WACZ itself, without access to our database, whether the WACZ was itself produced by Scoop, or was packaged up after the fact... which could be useful/good to have

I'll add to a list of things to talk about one more time, if we choose to convert the corpus in earnest.

Reply:
> It's a matter of preference,

Oh cool! I... If at any point I knew that licit syntax, I had forgotten. Will definitely tweak.

Reply:
Re: WARC filename, that makes sense. I misunderstood the inclusion of `{link.guid}.warc.gz` here as a new change (I'm used to seeing the output on the Scoop side). I think it's fine to keep doing what we're doing.

Reply:
It was a surprise to me! I wrote code to try and look at `data.warc.gz` and spent longer than I should have mystified by why it wasn't working 🙄 🤣